### ISSUE TITLE
Why stepsecurity is refereneced so much?

### ISSUE THREAD

**Issue by anantshri** [ossf/scorecard#4081]
Hi Team,

I have run your tool and multiple results appear,  I was under the impression this is an open project with ossf spear heading. however lots of issues raised effectively ask me to give permission to step security to scan and give me results and fixes. 

earier step security use to show me the delta fix and i could manually copy paste. new interface is abstracting that away and asking for org permissions.

in light of this do you really feel we should keep having step security as the default option in this project.

I am opening this issue to understand the logics for this tool / company being supported by this project. 
Thanks for answering in advance.

**Comment by varunsh-coder** [comment]
Hi @anantshri, I am a cofounder of StepSecurity, and want to understand why you think the new experience is asking for org permissions?

When a scorecard issue (for token permissions or pinning of actions) points to app.stepsecurity.io, it goes to this UI, and here you can click the link (highlighted) to go to the old experience (where you can copy the fixed workflow). 

<img width="1073" alt="Screen Shot 2024-05-04 at 2 07 41 PM" src="https://github.com/ossf/scorecard/assets/25015917/54167c7b-5aef-4ca4-87b0-68dc97b96294">


The new experience fixes more issues (e.g. adding CodeQL, dependabot file etc), using a pull request, and so reduces the steps developers need to take. Even for this experience, no org permission is asked for. The UI asks to login to get public data only (screenshot below). This is to ensure developers can only create PR in repos they have already contributed to. 

<img width="597" alt="Screen Shot 2024-05-04 at 2 12 10 PM" src="https://github.com/ossf/scorecard/assets/25015917/f283e503-a079-4883-a927-3717ba8ab8b4">

So, would love to understand why you think it needs org permissions?

Also, JFYI, over 1,200 public repos have used and increased their scorecard scores using the new PR experience, which would otherwise have taken them a lot more effort.

**Comment by varunsh-coder** [comment]
Thanks for clarification and I believe we are on the same page regarding permissions - that the tool does not need org permissions, only access to public data. 

>> why are there multiple issues being raised

This is how typical scanners/ security tools work, to enable tracking each issue separately. Scorecard does publish remediation instructions with each issue, such that developers can fix it manually if they want to, without using automated remediation. Moreover, the StepSecurity tool (which is open source), can only remediate a subset of the issues. 

W.r.t the change in the UI experience, that was done based on feedback from the community, and was updated only after discussion and consensus in a Scorecard community call about a year ago. We have been working with Scorecard since early 2022 to help maintainers improve Scorecard scores through automation.
